schema.graphql

```diff
@@ -27,12 +27,16 @@ type Token @entity {
 type Order @entity {
   "Trade's OrderUid to hex string"
   id: ID!
-  "Trade's owner"
-  owner: User!
-  "block's timestamp"
-  timestamp: BigInt!
+  "Trade's owner or presign User"
```

Comment:
I would just say `Trader's account`

the presign user is the same as the owner and that is also the same as  the trader